MicroPython/src/peripherals/sensor.py

```diff
@@ -52,10 +52,14 @@ def measure(self) -> None:
         :return: None
         """
         logging.debug("Sensor.measure()")
-        try:
-            self.sensor.measure()
-            self.sensor_last_measure_status = True
-        except OSError:
+        for _ in range(3):
```

Comment:
Maybe add some short comment to let others know in the future why you're trying to do it three times.